### DISCUSSION TITLE
FileDialog - Ability to copy, cut, paste, delete files

### DISCUSSION THREAD

**Discussion by valkyrienyanko** [godotengine/godot-proposals#4470] [GUI]
I have created a mod loader with a "Open Mods Folder" button with the intention of allowing users to manage mods easily by copy pasting mods they forgot to put in the mods folder. Clicking on the button would spawn a FileDialog window. The problem is you can't right click on any of the files to copy, cut, delete, paste the mod folders inside the FileDialog window. 

![image](https://user-images.githubusercontent.com/6277739/166085068-1be282a6-1faf-4f5b-8867-85434d9a5acd.png)

Edit: Just found out this is a thing https://stackoverflow.com/questions/1746079/how-can-i-open-windows-explorer-to-a-certain-directory-from-within-a-wpf-app

**Comment by me2beats** [comment]
I often have several projects opened, and copy files from one to another. And I really think this would be handy if I could select files and copy them right in godot (RMB-copy files) and then switch to another project (editor) and RMB- paste files.
